lighthouse-core/gather/gatherers/styles.js

```diff
@@ -129,13 +133,17 @@ class Styles extends Gatherer {
   afterPass(options) {
     return this.endStylesCollect(options.driver)
       .then(stylesheets => {
-        // Want unique stylesheets. Remove those with the same text content.
+        // Generally want unique stylesheets. Mark those with the same text content.
```

Comment:
I guess my question is, what audits are noisy? If you use flexbox in 100 style elements in all 100 of your shadow doms, shouldn't you have 100 failures? I couldn't find where else we use it.